web/html/src/manager/index.js

```diff
@@ -39,7 +39,7 @@ const pages = {
 }
 
 Object.keys(pages).forEach((key) => {
-  pages[key] = ["babel-polyfill/dist/polyfill", pages[key]];
+  pages[key] = ["@babel/polyfill/dist/polyfill", pages[key]];
```

Comment:
Can you make a quick test on old browsers if everything still works? I used a trial account on https://browserstack.com and tested it on a really old version of chrome

Reply:
Afaik we only support latest versions of major browsers. But anyway I'll try to run some tests.

Reply:
Hum could be interesting to validate if we need this polyfill at all, it would be less 100kb of downloaded and processed javascript on each page, I will create an issue for that